.github/workflows/pr-remove-request-review.yml

```diff
@@ -0,0 +1,25 @@
+name: Remove Review Label
+
+on:
+  pull_request_review:
+    types: [submitted, edited, dismissed]
+
+jobs:
+  remove-review-label:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    if: github.repository_owner == 'cryostatio' && contains(github.event.pull_request.labels.*.name, 'review-requested')
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Probably best to use the most recent version - it looks like `v4` just came out.